docs/en/customization/build.md

```diff
@@ -164,8 +162,8 @@ go env -w GONOSUMDB="gitlab/github.com/<company_name>/*"
 
 ::: tab Docker
 
-To streamline the build process, you can use the Docker image to build the application server and don't worry about
-installing Golang or any other dependencies on your local machine.
+You can simplify the build process by using the Docker image. It builds the application server and starts it
```

Comment:
It builds?
Velox builds the RoadRunner...
Velox doesn't start the RoadRunner after build, just puts it to the output folder specified by the user.